fenix/app/src/main/java/org/mozilla/fenix/components/metrics/MetricController.kt

```diff
@@ -275,6 +279,22 @@ internal class ReleaseMetricController(
             }
         }
 
+        Component.FEATURE_FXSUGGEST to FxSuggestFacts.Items.SPONSORED_SUGGESTION_CLICKED -> run {
+            val clickInfo = metadata?.get("clickInfo")?.let {
+                it as? FxSuggestClickInfo.Amp
+            } ?: return@run
+
+            FxSuggest.pingType.set("fxsuggest-click")
+            FxSuggest.source.set("urlbar")
```

Comment:
Per [doc](https://firefox-source-docs.mozilla.org/browser/urlbar/firefox-suggest-telemetry.html#click), `source` records:

> The source of the suggestion, either “remote-settings” or “merino”.

So it should be `remote-settings` here. Let's also update the description in metrics.yaml.

Reply:
removed base on https://github.com/mozilla-mobile/firefox-android/pull/3958#discussion_r1361039589